stable/artifactory-ha/templates/artifactory-installer-info.yaml

```diff
@@ -0,0 +1,25 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "artifactory-ha.fullname" . }}-installer-info
+  labels:
+    app: {{ template "artifactory-ha.name" . }}
+    chart: {{ template "artifactory-ha.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+data:
+  installer-info.json: |
+    {
+      "productId": "JFrogInstaller_Helm_{{ template "artifactory-ha.chart" . }}",
+      "features": [
+      {
+        "featureId": "AppVersion/{{ default .Chart.AppVersion .Values.artifactory.image.version }}"
+      },
+      {
+        "featureId": "Installer/{{ default "helm" .Values.installer.type }}"
+      },
+      {
+        "featureId": "Platform/{{ default "kubernetes" .Values.installer.platform }}"
```

Comment:
Why make this configurable?

Reply:
We will change it accordingly for the platforms like IKS, AKS, GKE, ICP etc